vg/Makefile

```diff
@@ -4,7 +4,7 @@ runtime_fullpath = $(realpath runtime)
 build_tool = runtime-container.DONE
 git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../vg | cut -f1 -d " ")
 name = quay.io/ucsc_cgl/vg
-tag = 1.4.0-cb68327--${git_commit}
+tag = 1.4.0-956-g677c443
```

Comment:
I think I already mentioned this, but `--${git_commit}` should occur at the end to make the tag less prone to being overwritten and thus reduce reproducibility. I can explain the motivation in person if necessary.